boards/shared/Src/Io/Io_SharedErrorTable.c

```diff
@@ -4,110 +4,105 @@
     SET_ERROR(
-        error_table, DIM_NON_CRITICAL_STACK_WATERMARK_ABOVE_THRESHOLD_TASKCANTX,
+        error_table, DIM_WARNING_STACK_WATERMARK_ABOVE_THRESHOLD_TASKCANTX,
         data->stack_watermark_above_threshold_taskcantx);
-    SET_ERROR(error_table, DIM_NON_CRITICAL_WATCHDOG_TIMEOUT, data->watchdog_timeout);
+    SET_ERROR(error_table, DIM_WARNING_WATCHDOG_TIMEOUT, data->watchdog_timeout);
 }
 
-static void
-    Io_ProcessFsmNonCriticalErrorMsg(struct ErrorTable *error_table, struct CanMsgs_fsm_non_critical_errors_t *data)
+static void Io_ProcessFsmNonCriticalErrorMsg(struct ErrorTable *error_table, struct CanMsgs_fsm_warnings_t *data)
 {
-    SET_ERROR(error_table, FSM_NON_CRITICAL_PAPPS_OUT_OF_RANGE, data->papps_out_of_range);
-    SET_ERROR(error_table, FSM_NON_CRITICAL_SAPPS_OUT_OF_RANGE, data->sapps_out_of_range);
+    SET_ERROR(error_table, FSM_WARNING_PAPPS_OUT_OF_RANGE, data->papps_out_of_range);
+    SET_ERROR(error_table, FSM_WARNING_SAPPS_OUT_OF_RANGE, data->sapps_out_of_range);
     SET_ERROR(
-        error_table, FSM_NON_CRITICAL_STACK_WATERMARK_ABOVE_THRESHOLD_TASK1HZ,
+        error_table, FSM_WARNING_STACK_WATERMARK_ABOVE_THRESHOLD_TASK1HZ,
         data->stack_watermark_above_threshold_task1_hz);
     SET_ERROR(
-        error_table, FSM_NON_CRITICAL_STACK_WATERMARK_ABOVE_THRESHOLD_TASK1KHZ,
+        error_table, FSM_WARNING_STACK_WATERMARK_ABOVE_THRESHOLD_TASK1KHZ,
         data->stack_watermark_above_threshold_task1_khz);
     SET_ERROR(
-        error_table, FSM_NON_CRITICAL_STACK_WATERMARK_ABOVE_THRESHOLD_TASKCANRX,
+        error_table, FSM_WARNING_STACK_WATERMARK_ABOVE_THRESHOLD_TASKCANRX,
         data->stack_watermark_above_threshold_taskcanrx);
     SET_ERROR(
-        error_table, FSM_NON_CRITICAL_STACK_WATERMARK_ABOVE_THRESHOLD_TASKCANTX,
+        error_table, FSM_WARNING_STACK_WATERMARK_ABOVE_THRESHOLD_TASKCANTX,
         data->stack_watermark_above_threshold_taskcantx);
-    SET_ERROR(error_table, FSM_NON_CRITICAL_WATCHDOG_TIMEOUT, data->watchdog_timeout);
-    SET_ERROR(error_table, FSM_NON_CRITICAL_BSPD_FAULT, data->bspd_fault);
-    SET_ERROR(error_table, FSM_NON_CRITICAL_LEFT_WHEEL_SPEED_OUT_OF_RANGE, data->left_wheel_speed_out_of_range);
-    SET_ERROR(error_table, FSM_NON_CRITICAL_RIGHT_WHEEL_SPEED_OUT_OF_RANGE, data->right_wheel_speed_out_of_range);
-    SET_ERROR(error_table, FSM_NON_CRITICAL_FLOW_RATE_OUT_OF_RANGE, data->flow_rate_out_of_range);
-    SET_ERROR(error_table, FSM_NON_CRITICAL_STEERING_ANGLE_OUT_OF_RANGE, data->steering_angle_out_of_range);
-    SET_ERROR(error_table, FSM_NON_CRITICAL_BRAKE_PRESSURE_OUT_OF_RANGE, data->brake_pressure_out_of_range);
-    SET_ERROR(error_table, FSM_NON_CRITICAL_BRAKE_PRESSURE_OPEN_OC, data->brake_pressure_open_oc);
-    SET_ERROR(error_table, FSM_NON_CRITICAL_BRAKE_PRESSURE_OPEN_SC, data->brake_pressure_open_sc);
+    SET_ERROR(error_table, FSM_WARNING_WATCHDOG_TIMEOUT, data->watchdog_timeout);
```

Comment:
with reference to my earlier comment, I think this reads nicely

Reply:
Its definitely more concise